.github/workflows/smoke-test.yaml

```diff
@@ -139,6 +133,7 @@ jobs:
              --set global.externalIPs.enabled=true \
              --set global.nodePort.enabled=true \
              --set global.hostPort.enabled=true \
+             --set config.bpfMasquerade=false \
```

Comment:
:+1:

Reply:
nit: this one should be added in https://docs.cilium.io/en/latest/gettingstarted/kind as well.

Reply:
I've noted that down on issue #12699 to follow up.